### DISCUSSION TITLE
Suppress Inside Temperature and Humidity in Kiosk Mode

### DISCUSSION THREAD

**Discussion by RonV42** [poblabs/weewx-belchertown#872] [General]
Just a quick question.  I am planning to use the kiosk page on a android tablet and my current weather data doesn't support indoor readings.  How does one suppress the Inside Temp and In Humid items from the page?

**Comment by ThePeePs** [comment]
In your config for the skin, under `Extras` set `aqi_enabled_kiosk` equal to 1.  This will replace the indoor temp with AQI from Aeris.  The rest of the config for the kiosk page can be found in the [README.md](https://github.com/poblabs/weewx-belchertown#kiosk-page)

**Reply by RonV42** [reply]
Thanks for the confg item that will replace temp/humidity with air quality.  I was poking around in the html file and kept messing up the references to the data that align the table elements.   AQI is a good solution.
